packages/object/published-types/object-test.ts

```diff
@@ -27,6 +28,8 @@ export default class UserProfileComponent extends EmberObject {
     const [first, last] = value.split(' ');
     this.setProperties({ first, last });
   }
+
+  @wrapComputed(emberComputed(() => {})) test: any;
```

Comment:
could you add a test that ensures non-any types pass through the `wrapComputed` function properly?

Reply:
oh also, please add some tests involving retrieving the value of the computed property generated by `@wrapComputed`. `ComputedProperty` is a really sensitive and tricky area of `@types/ember`, due to the need to simultaneously support the full matrix of:

* Ember.Object.extend
* class extends Ember.Object
* this.get('foo')
* this.foo

I'd like to ensure that changes to `@types/ember` that could break your consumers -- either in defining CPs, wrapping them or consuming them --  will be picked up in this lib's test suite.

Reply:
> I did add an additional test below in classic class syntax to make sure that wrapComputed does not interfere with getters/setters, I think that should be sufficient for now.

LGTM